src/shared_modules/utils/tests/socketDBWrapper_test.cpp

```diff
@@ -14,19 +14,17 @@
 #include "socketDBWrapper.hpp"
 
 // temp header
-#include "wazuhDBQueryBuilder.hpp"
 #include <chrono>
 #include <thread>
 
 TEST_F(SocketDBWrapperTest, EmptyTest)
 {
     m_query = "SELECT * FROM test_table;";
-    m_responses = std::vector<std::string> {""};
+    m_responses = std::vector<std::string> {" "};
 
     nlohmann::json output;
-    SocketDBWrapper socketDBWrapper(TEST_SOCKET);
     // The exception captured here is the timeout
```

Comment:
Update comment, we no longer have timeouts

Reply:
Done :)